### PR TITLE
Add sdf12 to ABI checker

### PR DESCRIPTION
**Pull request by azeey**
Needed by https://github.com/ignitionrobotics/sdformat/pull/724

### CODE REVIEW

File: jenkins-scripts/dsl/sdformat.dsl [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import _configs_.*
 import javaposse.jobdsl.dsl.Job
 
-def sdformat_supported_branches = [ 'sdformat6' , 'sdformat9', 'sdformat10', 'sdformat11' ]
-def sdformat_gz11_branches = [ 'sdformat9', 'sdformat10', 'sdformat11', 'main' ]
+def sdformat_supported_branches = [ 'sdformat6' , 'sdformat9', 'sdformat10', 'sdformat11', 'sdformat12' ]
+def sdformat_gz11_branches = [ 'sdformat9', 'sdformat10', 'sdformat11', 'sdformat12, 'main' ]
 // nightly and prereleases
 def extra_sdformat_debbuilder = [ 'sdformat12' ]
```
Comment (azeey) [inline jenkins-scripts/dsl/sdformat.dsl:7]: I don't know if we're supposed to remove `sdformat12` from `extra_sdformat_debbuilder`  or not.

Comment (scpeters) [reply jenkins-scripts/dsl/sdformat.dsl:7]: I think it could be removed

File: jenkins-scripts/dsl/sdformat.dsl [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import _configs_.*
 import javaposse.jobdsl.dsl.Job
 
-def sdformat_supported_branches = [ 'sdformat6' , 'sdformat9', 'sdformat10', 'sdformat11' ]
-def sdformat_gz11_branches = [ 'sdformat9', 'sdformat10', 'sdformat11', 'main' ]
+def sdformat_supported_branches = [ 'sdformat6' , 'sdformat9', 'sdformat10', 'sdformat11', 'sdformat12' ]
+def sdformat_gz11_branches = [ 'sdformat9', 'sdformat10', 'sdformat11', 'sdformat12, 'main' ]
```
Comment (scpeters) [inline jenkins-scripts/dsl/sdformat.dsl:5]: missing a closing `'`

Comment (scpeters) [reply jenkins-scripts/dsl/sdformat.dsl:5]: [Build Status](https://build.osrfoundation.org/job/_dsl_sdformat/1053/) https://build.osrfoundation.org/job/_dsl_sdformat/1053/console

File: jenkins-scripts/dsl/sdformat.dsl (hunk 1 above)
Comment (azeey) [reply jenkins-scripts/dsl/sdformat.dsl]: Ok, removed in https://github.com/ignition-tooling/release-tools/pull/525/commits/ec2431219fb337e5d3428773effb22feafaa4761

File: jenkins-scripts/dsl/sdformat.dsl (hunk 2 above)
Comment (azeey) [reply jenkins-scripts/dsl/sdformat.dsl]: Thanks. https://github.com/ignition-tooling/release-tools/pull/525/commits/ec2431219fb337e5d3428773effb22feafaa4761

**Comment by scpeters** [comment]
[Build Status](https://build.osrfoundation.org/job/_dsl_sdformat/1054/) https://build.osrfoundation.org/job/_dsl_sdformat/1054/

**Review by scpeters — APPROVED** [review]

**Pull request merged**